packages/asset-swapper/src/utils/quote_requestor.ts

```diff
@@ -57,17 +62,15 @@ export class QuoteRequestor {
                     params: {
                         sellToken,
                         buyToken,
-                        buyAmount: marketOperation === MarketOperation.Buy ? assetFillAmount.toString() : undefined,
-                        sellAmount: marketOperation === MarketOperation.Sell ? assetFillAmount.toString() : undefined,
+                        sellAmount: assetFillAmount.toString(),
                         takerAddress,
                     },
                     timeout: makerEndpointMaxResponseTimeMs,
                 }).catch(err => {
                     logUtils.warn(
-                        `Failed to get RFQ-T quote from market maker endpoint ${rfqtMakerEndpoint} for API key ${takerApiKey} for taker address ${takerAddress}: ${JSON.stringify(
-                            err,
-                        )}`,
+                        `Failed to get RFQ-T quote from market maker endpoint ${rfqtMakerEndpoint} for API key ${takerApiKey} for taker address ${takerAddress}`,
                     );
+                    logUtils.warn(err);
```

Comment:
`JSON.stringify(err)` was just outputting `{}` so I needed to add this to see the actual error